apps/src/blockly/types.ts

```diff
@@ -81,10 +81,18 @@ export interface SerializedFields {
   };
 }
 
+interface AnalyticsData {
+  appType: string;
+  scriptName: string;
+  scriptId: number;
+  levelId: number;
```

Comment:
Should some of these be considered optional since they're not provided for Music Lab?